src/state-incentive-details.tsx

```diff
@@ -463,83 +471,61 @@ type Props = {
  * currently selected.
  */
 export const StateIncentives: FC<Props> = ({
-  firstResultsRef,
-  secondResultsRef,
+  resultsRef,
   response,
-  selectedProjects,
   emailSubmitter,
 }) => {
   const { msg } = useTranslated();
 
-  // We're filtering out IRA rebates in favor of state-specific handling.
-  const allEligible = response.incentives
-    // Forward compatibility for when only eligible incentives are returned from
-    // the API, with no eligible flag.
-    .filter(i => i.eligible === true || i.eligible === undefined)
-    .filter(i => !isIRARebate(i));
```

Comment:
This filtering is all being done in the backend now; we're not losing any functionality here.